index.html

```diff
@@ -39,14 +43,52 @@
         <p class="menu__phone__title uppercase">Telefon całodobowy</p>
         <a href="tel:+48603670456" class="menu__phone__number">+48 603 670 456</a>
       </div>
+
+
+      <button class="ham">
+        <!-- material icons https://material.io/resources/icons/ -->
+        <span class="menuIcon material-icons">
+          menu
+        </span>
+        <span class="xIcon material-icons">
+          close
+        </span>
+      </button>
+      <ul class="menu-mobile">
+        <li><a class="menuLink" href="#">Strona główna</a></li>
+        <li><a class="menuLink" href="#services">Usługi pogrzebowe</a></li>
+        <li><a class="menuLink" href="documents">Dokumenty</a></li>
+        <li><a class="menuLink" href="#">Mówca pogrzebowy</a></li>
+        <li><a class="menuLink" href="#contact">Kontakt</a></li>
+      </ul>
+
```

Comment:
Seems we have to many empty lines here. :)

Reply:
I removed it manually, Beautify doesn't help on this point.

Reply:
OK, I see, whatever you prefer. :)